apps/desktop/app/process/AutoUpdate.ts

```
@@ -541,10 +541,13 @@ const init = ({ mainWindow, store }: IDependencies) => {
     await clearUpdateCache();
   });
 
-  ipcMain.on(ipcMessageKeys.UPDATE_SETTINGS, (_, settings: IUpdateSettings) => {
-    logger.info('auto-update', 'Set setting: ', JSON.stringify(settings));
-    setUseTestFeedUrl((settings ?? {}).useTestFeedUrl ?? false);
-  });
+  ipcMain.on(
+    ipcMessageKeys.UPDATE_SETTINGS,
+    (_, settings: IDesktopStoreUpdateSettings) => {
+      logger.info('auto-update', 'Set setting: ', JSON.stringify(settings));
+      setUseTestFeedUrl((settings ?? {}).useTestFeedUrl ?? false);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)